frontend/libs/studio-components/src/storybook-utils/decorators/FixedWidthDecorator/FixedWidthDecorator.tsx

```
@@ -0,0 +1,9 @@
+import type { PropsWithChildren } from 'react';
+import React from 'react';
+import classes from './FixedWidthDecorator.module.css';
+
+export type FixedWidthDecoratorProps = PropsWithChildren<{}>;
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)